pkg/envtest/webhook.go

```diff
@@ -179,10 +180,6 @@ func (o *WebhookInstallOptions) PrepWithoutInstalling() error {
 
 // Install installs specified webhooks to the API server
 func (o *WebhookInstallOptions) Install(config *rest.Config) error {
-	if err := o.PrepWithoutInstalling(); err != nil {
```

Comment:
maybe make this call conditional so we don't break people if anyone's using it manually

Reply:
(i.e. don't re-call if we've already called it, but if someone's using it standalone still call)

Reply:
if it's not too complicated